_projects/meta_review.md

```diff
@@ -0,0 +1,57 @@
+---
+collaborating_projects:
+  - coala
+desc: "Implement a meta-review system that tracks and provides meaningful analysis of reactions on reviews."
+difficulty: medium
+initiatives:
+  - GSoC
+issues:
+  - "https://github.com/coala/projects/issues/480"
+markdown: meta_review.md
+mentors:
+  - hemangsk
+  - blazeu
+  - alisianoi
+  - andrewda
+name: "Meta-review"
+requirements:
+  - "The applicant should have at least one patch merged to any of the coala repositories."
+  - "The applicant should be familiar with GitHub GraphQL API and javascript."
+status: in_progress
+tags:
+  - community
+  - web
+---
+
+The goal of project meta-review is to build a meta-review track system similar to a meta-moderation system that can track emoji responses to review comments. These responses are to be collected, processed and displayed on [gh-board][https://github.com/coala/gh-board], which is a nice serverless kanban board.
+
+People including the author of pull request respond to comments by attaching emojis. Those emojis are called meta-review, or [reactions][https://developer.github.com/v4/enum/reactioncontent/]. THUMBS_UP and THUMBS_DOWN together with other reactions are used by people especially the author of pull request to publicly acknowledge or reject review comments. This provides feedback to reviewers and all other members.
+
+However, those reactions spread over all PRs and are not collected and analyzed. By tracking reactions, a nice feedback loop would be created for both senior and junior reviewers. Also, this information can be used as newcomer metrics.
+
+#### Milestones
+
+##### PREPARATION/BONDING
+
+- The applicant has proposed and merged a cEP of how meta-reviews could be tracked and handled.
+- The applicant has created mockups for the frontend.
+- The applicant gets acquainted with materialize CSS and data visualization library (e.g. d3.js).
+- The applicant probably gets [gh-board#13][https://github.com/coala/gh-board/issues/13] fixed and refactors React code a bit.
+
+##### CODING PHASE 1
+
+- A basic gh-board that displays reaction analysis exists.
```

Comment:
"analysis" is not mentioned in the project description above.
As a result it is not clear what will be delivered in this phase.